contracts/IERC20.sol

```diff
@@ -13,13 +13,13 @@ interface IERC20Events {
      *
      * Note that `value` may be zero.
      */
-    event Transfer(address indexed from, address indexed to, uint256 amount);
+    event ERC20Transfer(address indexed from, address indexed to, uint256 amount);
```

Comment:
we can't change the event signature (its name in this case) as this changes the ABI.